packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -416,6 +416,7 @@ const getAxesScale = (axis, scale, config) => {
 
   axis.x = prepareXAxis(
     scale.x,
+    // undefined,
```

Comment:
Is this still needed?

Reply:
fixed here: 7c7d7cbed6dfcaec72005722f5448bf39aa3fbff